index.html

```diff
@@ -0,0 +1,212 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My Portfolio</title>
+  <link rel="connect" href="InaiMathi.ttf">
+  <link rel="preconnect" href="https://fonts.gstatic.com"> 
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@100;200;300;400;500;700&family=Poppins:wght@100;200;300;400;500&family=Roboto:wght@100;300;400;500;700&display=swap" rel="stylesheet">
+  <link
+    rel="stylesheet"
+    href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css"
+    type="text/css"
+  />
+  <link href="styles.css" rel="stylesheet" />
+</head>
+<body>
+  <header>
+    <nav class="header">
+      <a href="#" class="log">Kaneez Fatima</a>
+      <a><i class="fa fa-bars cursor"></i></a>
+    </nav>
+  </header>
+      <section class="main-section img">
+        <div>
+          <h1>Hey There.</h1>
+            <h1>I'm Kaneez Fatima</h1>
+            <h2>I'am a Software Developer</h2>
+          
+    
+          <p class="para">
+            I can help you build a product, feature or website. Look through some
+            of my work and experience! If you like what you see and have a project
+            you need coded, don't hestiate to contact me.
+          </p>
+    
+          <ul>
+         <li class="inline"><a
+            href="https://github.com/Kaneez98Fatima"
+            target="_blank" rel="noopener noreferrer"><i class="fa fa-github icon"></i></a></li>
+          <li class="inline"><a
+            href="https://www.linkedin.com/in/kaneez-fatima-0a86601b3"
+            target="_blank"
+            rel="noopener noreferrer "
+            ><i class="fa fa-linkedin icon"></i
+          ></a></li>
+         <li class="inline"> <a
+            href="https://www.instagram.com/fatima98kaneez."
+            target="_blank"
+            rel="noopener noreferrer"
+            ><i class="fa fa-instagram icon"></i></a></li>
+             <li class="inline"><a href="Mail:Kaneez98fatima@gmail.com" target="_blank" rel="noopener"
+                ><i class="fa fa-envelope icon"></i
+              ></a></li>
+            </ul>
+            </div>
+        </section>
```

Comment:
As the previous reviewer stated you should stick to the design template as much as possible this is how this section looks 
![Screenshot (59)](https://user-images.githubusercontent.com/52497246/121771441-513db280-cb67-11eb-8702-ead1c601e2a3.png)

This is the original design
![Screenshot (62)](https://user-images.githubusercontent.com/52497246/121771453-61ee2880-cb67-11eb-85aa-78ef6afe3316.png)